pkg/virt-controller/services/template_test.go

```diff
@@ -3954,6 +3954,20 @@ var _ = Describe("Template", func() {
 			}))
 		})
 
+		It("should compute the correct tolerations when rendering hotplug attachment pods", func() {
+			vmi := api.NewMinimalVMI("fake-vmi")
+			vmi.Spec.Tolerations = append(vmi.Spec.Tolerations, k8sv1.Toleration{Key: "test"})
+			ownerPod, err := svc.RenderLaunchManifest(vmi)
+			Expect(err).ToNot(HaveOccurred())
+
+			vmi.Status.SelinuxContext = "test_u:test_r:test_t:s0"
+			claimMap := map[string]*k8sv1.PersistentVolumeClaim{}
+			pod, err := svc.RenderHotplugAttachmentPodTemplate([]*v1.Volume{}, ownerPod, vmi, claimMap)
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(pod.Spec.Tolerations).To(BeEquivalentTo(ownerPod.Spec.Tolerations))
```

Comment:
I believe what you're really after is testing that ```RenderHotplugAttachmentPodTemplate().Tolerations == vmi.Spec.Tolerations```. That would keep the test more honest in case something lied about the ownerPod (virt-launcher associated with VMI)

Could also perform same check for RenderHotplugAttachmentTriggerPodTemplate though I am not sure we still use that. @awels do you happen to know?

Reply:
You're right, I'll do that.

I'll also add the same check for `RenderHotplugAttachmentTriggerPodTemplate`.